app/app.js

```diff
@@ -42,8 +48,20 @@ function populateTemperFilter (chickenTemper){
   }
 }
 
-allChickens.forEach(function(a){
-  $('#chickens').append(a.toHtml());
-  populateChickenFilter(a.title);
-  populateTemperFilter(a.temperament);
-});
+
+Chicken.fetchAll = function() {
+  if (localStorage.rawChickenData) {
+    Chicken.loadAll(JSON.parse(localStorage.rawChickenData));
+  } else {
+    $.getJSON( 'data/appData.json', function ( json ) {
+      console.log( 'JSON Data: ', json);
+      localStorage.setItem('rawChickenData', JSON.stringify(json));
+    });
+  }
+  view.initIndexPage();
+}
+
+
+// $.getJSON('app/appData.json', function(json){
+//   console.log(json);
+// });
```

Comment:
Any commented code should be removed before doing a PR, to keep things clean and less distracting.